CliFx/Domain/CommandArgumentSchema.cs

```diff
@@ -18,6 +18,37 @@ internal abstract partial class CommandArgumentSchema
 
         public bool IsScalar => TryGetEnumerableArgumentUnderlyingType() == null;
 
+        public IEnumerable<string> GetValidValues()
+        {
+
+            var propertyType = Property?.PropertyType;
+
+            // Property can actually be null here due to damn it operator in lines 103 and 106,
+            // so we have to check for now. In such case, let's end early.
+            if (propertyType is null)
+            {
+                yield break;
+            }
+
+            // Handle nullable values.
+            // If the property is nullable, this will return a non-null value.
+            var underlyingType = Nullable.GetUnderlyingType(propertyType);
+            if (underlyingType is object && propertyType.IsValueType)
+            {
+                yield return "null";
```

Comment:
I think we don't need to return `null` as valid value, because the user can't actually specify `null`, they instead just not specify the option instead. So let's just have the same behavior for nullable and non-nullable enums where it just returns the list of fields.

Reply:
Fixed.